py/runtime.c

```diff
@@ -184,6 +184,15 @@ mp_obj_t mp_load_global(qstr qst) {
         #endif
         elem = mp_map_lookup((mp_map_t*)&mp_module_builtins_globals.map, MP_OBJ_NEW_QSTR(qst), MP_MAP_LOOKUP);
         if (elem == NULL) {
+            #if MICROPY_LAZY_LOAD_GLOBAL
+            // allow port to provide this global
+            mp_obj_t obj = mp_lazy_load_global(qst);
+            if (MP_OBJ_NULL != obj) {
```

Comment:
For consistency use `obj ~= MP_OBJ_NULL`

Reply:
Yep